### DISCUSSION TITLE
Firebase Blaze charging daily when not using Rowy?

### DISCUSSION THREAD

**Discussion by njproductionsus** [AntlerVC/firetable#1539] [Support / Q&A]
I followed the Rowy guide to build Parallel GPT: https://www.rowy.io/blog/parallelgpt

It works GREAT! However I do not use it often, maybe once a month for a day. I am noticing when it is NOT in use, that Firebase is still charging me a flat daily fee of $0.75 - $1.25/day. I can't figure out if this is correct or if somehow Rowy is running daily even when I'm not using it.

Is the Firebase BLAZE plan required (as Firebase support said I should downgrade to Firebase Spark)? But I wasn't clear if that would loose my setup (aka I can't just switch between the two as needed)?

![image](https://github.com/rowyio/rowy/assets/25138672/84ec71cc-955f-4a93-9047-668d89ac83b4)

**Comment by bofeiw** [comment]
Firebase gives you a generous amount of free usage before you are charged. When you are charged, it generally mean that you have exceeded the free usage according to their [pricing details](https://firebase.google.com/pricing). Since you are charged the same every day, I would check if you are storing a lot of files in Firebase Storage as it is charged at a flat rate (e.g. $0.026/GB).

However, to have a full picture understanding of your billing, you can generate a [GCP billing report](https://cloud.google.com/billing/docs/how-to/reports). With the report, you can see exactly which service is costing you and take actions further.

To answer your second question, yes you can switch between Firebase Spark Plan and Blaze Plan as needed. But be aware that any features that requires Cloud Functions in Rowy will stop working when your billing is disabled.

**Reply by njproductionsus** [reply]
The project in question was built exclusively for this Rowy Parallel GPT instance. It has not been used by me in a month and still incurs those fees. SO my question is to try and resolve why the Rowy Parallel GPT setup results in daily fees if not in use.

Unfortunately the GCP report was too vague, just referencing the cost was from an external service.
